### DISCUSSION TITLE
Low resolution chart display

### DISCUSSION THREAD

**Discussion by imotep33** [grafana/grafana#44352] [Dashboards, Panels & Visualizations]
Hi,
I'm using Grafana with Home assistant to display a kind of weather station.
I've set some nice graphs to analyse weather and it looks cool on my computer. However when I display it on my smartphone, the graphs are not really efficient. my screen resolution is 1136 × 640, but it could also be a raspberry pi screen which have roughly the same resolution.
The legend and the text are taking too much space around the chart itself. The result is I have a chart where i can see the evolution, but it's not precise enough to read properly the values.
It would be better if there were "compact" feature when sharing the embedded link of the panel. this could bring more lines on the X and Y axis to enhance reading.

A second thing is the time axis. The unit is either hourly based or a date+time. if you want to have a 36h or 48h history it significantly reduces the vertical lines on the X axis because all the values are "dd/mm hh:mm"
for this kind of time span, Home assistant has a very smart way of showing it. It shows only the hours, except for the 00:00 time, it's replaced by the date as "dd/mm" with bold characters. its allows to double the X axis displayed value and makes the chart easyer to read.

enclosed, you can the embedded link of my time charts.
here the Y axis is not enough detailed.
[https://i.postimg.cc/90Y0TZd0/IMG-7756.png](url)

here the X axis has only one value.
[https://i.postimg.cc/fWcJ6nGs/IMG-7757.png](url)

and here is the home assistant chart on the same device with a time base as I described.
[https://i.postimg.cc/GtC9Hqs6/IMG-7755.png](url)

Sorry if my explanations are not very clear, but i'm not native english speaker.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
